internal/publish/publish.go

```diff
@@ -253,11 +253,6 @@ func (p *defaultPublisher) doPublish() error {
 		return err
 	}
 
-	err = p.configureInterpreters()
-	if err != nil {
-		return err
-	}
```

Comment:
Do we need to configure interpreters prior to creating the manifest? I see that `createManifest` calls `getRPackages` which uses the interpreters.

Further look into `readInterpreterDetails` it doesn't look like `getRPackages` depends on the changes to `defaultPublisher.Target` that it sets.

Just thinking through this out loud to make sure I'm not missing something.